src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/tools/GoldPan.java

```diff
@@ -59,14 +63,31 @@ public GoldPan(ItemGroup itemGroup, SlimefunItemStack item, RecipeType recipeTyp
     }
 
     /**
-     * This method returns the target {@link Material} for this {@link GoldPan}.
-     * 
-     * @return The {@link Material} this {@link GoldPan} can be used on
+     * @deprecated
+     *
+     * This method returns the old {@link Material} that this {@link GoldPan} accepted
+     *
+     * @return The old {@link Material} this {@link GoldPan} could be used on
```

Comment:
i get what you mean but i read this literal you dont return the old material

Reply:
I don't think I follow

Reply:
Remove the old words. You are implying that those are old materials even though they arent old.

Reply:
Maybe "legacy" is a better word? It's a method that I marked for removal anyway on the GH project (or that I should mark for removal)

Reply:
I mean it’s not legacy either to method stays the same and the material isn’t legacy docs doesn’t need to be changed when it’s marked as deprecated

Reply:
Aaah I see what you mean. Yeah ok makes sense to me now

Reply:
Got it, I’ll change the wording in a bit then, can’t atm